content/_data/changelogs/weekly.yml

```diff
@@ -915,6 +915,47 @@
     # pull: 2960 not notable enough
     # pull: 2963 not notable enough
     # pull: 2964 not notable enough
+- version: "2.74"
+  date: 2017-08-13
+  changes:
+    - type: major rfe
+      message: >
+        Upgrade Stapler library from 1.250 to 1.252
+      references:
+        - pull: 2956
```

Comment:
Not sure it is enough. I would mention..
* Bug-fix: JENKINS-45903:
* Internal(?) bugfix: JENKINS-43715, JENKINS-40979